build.sbt

```diff
@@ -3,7 +3,7 @@ import sbtrelease.ReleasePlugin._
 
 name := "cromwell"
 
-version := "0.12"
+version := "0.13"
```

Comment:
Argh the real problem is this PR is against master and not develop.  :frowning:  Closing and reopening...